spec/features/hiring_staff_can_publish_a_vacancy_spec.rb

```diff
@@ -150,11 +150,11 @@
           expect(page).to have_content(I18n.t('activerecord.errors.models.vacancy.attributes.contact_email.blank'))
         end
 
-        within_row_for(element: 'legend', text: I18n.t('jobs.deadline_date')) do
+        within_row_for(element: 'span', text: I18n.t('jobs.deadline_date')) do
           expect(page).to have_content(I18n.t('activerecord.errors.models.vacancy.attributes.expires_on.blank'))
         end
 
-        within_row_for(element: 'legend', text: I18n.t('jobs.publication_date')) do
+        within_row_for(element: 'span', text: I18n.t('jobs.publication_date')) do
```

Comment:
Does the gem now generate a `fieldset` without a `legend`?  This doesn't sound great.  I'm not aware of how a screen-reader would react in this scenario, but it feels dodgy.

Reply:
Thanks.
The fieldset does have a legend. 
However, after comparing the forms on development and staging, have realised that what should have been the legend text was written as hint text instead. My mistake!